src/components/ActionableInput/index.tsx

```diff
@@ -48,7 +49,8 @@ const ActionableInputContainer = styled.div`
 
 const ButtonContainer = styled(Button)<{ $wasClicked?: boolean }>`
   padding: 8px;
-  margin-left: 4px;
+  margin-left: -34px;
+  z-index: 2;
```

Comment:
what's this needed for?

Reply:
the input is `disabled` in `aserto-console` so these buttons weren't clickable once they were moved

Reply:
setting `pointer-events: none` on the input should let them make it through to the buttons